src/lib/ui/thermostat/ClimateSettingScheduleDevice.tsx

```diff
@@ -0,0 +1,54 @@
+import { isThermostatDevice, type ThermostatDevice } from 'seamapi'
+
+import { ChevronRightIcon } from 'lib/icons/ChevronRight.js'
+import { useDevice } from 'lib/seam/devices/use-device.js'
+import { DeviceImage } from 'lib/ui/device/DeviceImage.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+export function ClimateSettingDevice({
+  deviceId,
+}: {
+  deviceId: string
+}): JSX.Element | null {
+  const { isLoading, device } = useDevice({
+    device_id: deviceId,
+  })
+
+  if (isLoading) {
+    return null
+  }
+
+  if (device == null) {
+    return null
+  }
+
+  if (!isThermostatDevice(device)) {
+    return null
+  }
+
+  return <Content device={device} />
+}
+
+function Content(props: { device: ThermostatDevice }): JSX.Element {
+  const { device } = props
+
+  return (
+    <div className='seam-climate-setting-schedule-device-bar'>
+      <div className='seam-climate-setting-schedule-device-content'>
+        <div className='seam-device-image'>
+          <DeviceImage device={device} />
+        </div>
+        <div className='seam-climate-setting-schedule-device-details'>
+          <div className='seam-device-name'>{device.properties.name}</div>
+          <div className='seam-device-current-climate-setting'>
+            Current climate:
+            <ClimateSettingStatus
+              climateSetting={device.properties.current_climate_setting}
+            />
+          </div>
+        </div>
+      </div>
+      <ChevronRightIcon className='chevron' />
```

Comment:
Cannot have any non prefixed classnames.

Reply:
Oops, that's old.

Reply:
https://github.com/seamapi/react/pull/378/commits/dd76de64d418e71e13b2152e1c6e37b43a7c91c5#diff-6a5322f6b608576116fc0fa481fbe912bdc8640f9ebf73bc7ea343b0216a79d9R51